common/v2/features/BuyAssets/BuyAssetsForm.tsx

```diff
@@ -0,0 +1,199 @@
+          console.debug('err detected: ', err);
+          window.open(`${BUY_MYCRYPTO_WEBSITE}`, '_blank');
+        });
+    } else {
+      window.open(`${BUY_MYCRYPTO_WEBSITE}`, '_blank');
+    }
+  };
+
+  const goBack = () => history.push(ROUTE_PATHS.DASHBOARD.path);
+
+  return (
+    <ContentPanel
+      onBack={goBack}
+      backBtnText={translateRaw('DASHBOARD')}
+      heading={translateRaw('DASHBOARD_ACTIONS_BUY_TITLE')}
+    >
+      <Formik
+        initialValues={initialFormikValues}
+        validationSchema={BuyFormSchema}
+        onSubmit={vals => handleSubmission(vals, SubmissionType.SEND_TO_SELF)}
+        render={({ values, errors, touched, setFieldValue }) => {
+          const relevantAccounts = accounts.filter(account => {
+            return values.asset && values.asset.networkId
+              ? isAccountInNetwork(account, values.asset.networkId)
+              : true;
+          });
+
+          return (
+            <Form>
+              <FormFieldItem>
+                <FormFieldLabel htmlFor="account">{translate('X_RECIPIENT')}</FormFieldLabel>
+                <Field
+                  name="account"
+                  value={values.account}
+                  component={({ field }: FieldProps) => (
+                    <AccountDropdown
+                      name={field.name}
+                      value={field.value}
+                      accounts={relevantAccounts}
+                      onSelect={(option: IAccount) => {
+                        setFieldValue('account', option); //if this gets deleted, it no longer shows as selected on interface, would like to set only object keys that are needed instead of full object
+                      }}
+                    />
+                  )}
+                />
+                {errors && errors.account && touched && touched.account && (
+                  <InlineMessage className="SendAssetsForm-errors">{errors.asset}</InlineMessage>
+                )}
+              </FormFieldItem>
+              <FormFieldItem>
+                <FormFieldLabel htmlFor="asset">
+                  {translate('SEND_ASSETS_ASSET_SELECTION_PLACEHOLDER')}
+                </FormFieldLabel>
+                <Field
+                  name="asset"
+                  value={values.asset}
+                  component={({ field, form }: FieldProps) => (
+                    <AssetDropdown
+                      selectedAsset={field.value}
+                      assets={filteredAssets}
+                      onSelect={(option: Asset) => {
+                        form.setFieldValue('asset', option || {}); //if this gets deleted, it no longer shows as selected on interface (find way to not need this)
+                      }}
+                    />
+                  )}
+                />
+                {errors && errors.asset && touched && touched.asset ? (
+                  <InlineMessage className="SendAssetsForm-errors">{errors.asset}</InlineMessage>
+                ) : null}
```

Comment:
Same thing as it was already fixed (ternary operator can be removed), probably you missed my comment regarding the same problem in these lines too. :)

Reply:
woops